src/Http/Controllers/Web/ActivityController.php

```diff
@@ -18,11 +19,14 @@ public function __construct(private readonly ActivityRepository $activities)
      */
     public function index(Request $request): View
     {
-        $activities = $this->activities->paginateActivities(perPage: 20, search: $request->search);
+        $activities = $this->activities->paginateActivities(perPage: 20, search: $request->search, user_id: $request->user_id);
 
         return view('user-activity::index', [
             'adminView' => true,
             'activities' => $activities,
+            'selectedUser' => DB::table('users')
```

Comment:
@pajazi Let's import `Vanguard\Repositories\User\UserRepository` class here and use the `find(...)` method to get the user by id.